libs/platform/src/lib/components/object-attribute/object-attribute.component.html

```diff
@@ -0,0 +1,8 @@
+{{label}}
+<ng-content></ng-content>
```

Comment:
Why do we need `<ng-content>` here? Will it ever get used instead of `label` attribute? If so can you add an example in the documentation where `<ng-content>` is used.